.github/pr_issue_template.md

```diff
@@ -0,0 +1,45 @@
+## Related Issue 
+  - Info about the related issue 
+
+Closes: #[issue number that will be closed through this PR]
+
+#### Describe the changes you've made
+
+Give a clear description what modifications you have made
+
+## Type of change
+
+What sort of change have you made :
+<!--
+Example how to mark a checkbox :-
+- [x] My code follows the code style of this project.
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Code style update (formatting, local variables)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Describe how have you verified the changes made
+
+## Checklist:
+<!--
+Example how to mark a checkbox :-
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the guidelines of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly whereever it was hard to understand.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] My changes generate no new warnings.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] Any dependent changes have been merged and published in downstream modules.
```

Comment:
Just telling, when you were to accept all these, this you haven't done! Hehe. Do you see? Dependent changes merged? Your dependent change of Contributor Code of Conduct didn't get merged! 😞  So you have to technically uncheck this one! LoL.

Reply:
But as it is not yet merged it wont show up show are you asking me to remove this part for now?

Reply:
I was just telling you in a funny tone man... 😜

Reply:
Where's the line now? Did you remove it? 🤔